modules/createServerRenderContext.js

```diff
@@ -1,34 +1,36 @@
-const k = () => {}
-
 const createServerRenderContext = () => {
   let flushed = false
+  let flushedIndex = 0
   let redirect = null
   let matchContexts = [
-    /* { hasMissComponent: bool, matchesByIdentity: [] } */
+    /* { hasMissComponent: bool, hasMatches: bool } */
   ]
 
-  const setRedirect = flushed ? k : (location) => {
```

Comment:
I think it was probably there because the second render pass might still trigger that `setRedirect` function, so it is actually possible to trigger `k` (although it's unlikely to see a chain of redirects), and I agree you'd never use that `redirect` value again, so modifying doesn't actually affect anything.